budget-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+server.port=8080
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
Please align these configurations with the latest ones on develop.
You can use the user-service as an example